refs/biblio.json

```diff
@@ -1288,6 +1288,16 @@
     "HTMLMEDIACAPTURE": {
         "aliasOf": "html-media-capture"
     },
+    "HTTP-JFV": {
+        "authors": [
+            "J. Reschke",
```

Comment:
JSON doesn't support trailing commas in arrays. :-(

Mind fixing that?

Reply:
Whoops!  Done